### PR TITLE
Resolve issue #75 - Tag blocking is not working.

### PR DESCRIPTION
**Pull request by AvalancheYT**
Due it not working, the line ``` String vartag = outputTag.toLowerCase(); ``` did not work as it is not defined.
Merge this and the tag will be fixed. (Tested on my LAN server, working like normal)
@LegendIsAwesomes

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java [hunk 1]
```diff
@@ -125,12 +130,14 @@ else if ("set".equalsIgnoreCase(args[0]))
...
-                    String vartag = outputTag.toLowerCase();
-
-                    if (vartag.contains("admin") && vartag.contains("mod") && vartag.contains("owner") && vartag.contains("moderator") && vartag.contains("developer") && vartag.contains("console") && vartag.contains("super admin") && vartag.contains("telnet admin") && vartag.contains("senior admin"))
+                    for (String word : FORBIDDEN_WORDS)
                     {
-                        Command_smite.smite(sender_p, "[AutoSmite] You are not an administrator");
-                        return true;
+                        if (rawTag.contains(word))
+                        {
+                            playerMsg("That tag contains a forbidden word.");
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: May be better if it is like this :) `playerMsg("[AutoSmite] That tag contains a forbidden word.");`

Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: May be better if it is like this :) `playerMsg("[AutoSmite] That tag contains a forbidden word.");`

Comment (AvalancheYT) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: Added to smite part. https://github.com/ImmaFreedom-Dev/ImmaFreedomMod/pull/97/files#diff-57260c906c3221051fa5c1b4f543a4b8R138

**Comment by DUFCLiam** [comment]
@LegendIsAwesomes Looks like it works.

**Pull request merged**